src/universal/modules/meeting/components/CheckInCards/CheckInCards.js

```diff
@@ -6,34 +6,38 @@ import CheckInCard from 'universal/modules/meeting/components/CheckInCard/CheckI
 import checkInCardBaseStyles from '../CheckInCard/checkInCardBaseStyles';
 import {cashay} from 'cashay';
 
-const makeCheckinPressFactory = (teamMemberId) => (isCheckedIn) => () => {
+const makeCheckinPressFactory = (teamMemberId, gotoNext) => (isCheckedIn) => () => {
   const options = {
     variables: {
       isCheckedIn,
       teamMemberId
     }
   };
   cashay.mutate('checkIn', options);
+  gotoNext();
```

Comment:
yeah, it's 2 calls to the server, but websockets are great for short little messages.